### ISSUE TITLE
Concat Fields processor

### ISSUE THREAD

**Issue by nsoft** [nsoft/jesterj#55]
A more performance oriented alternative to FieldTemplateProcessor, that can join arbitrary number of fields, possibly with a delimiter.

**Comment by levimatheri** [comment]
Hi @nsoft. I'm assuming this is still your for grabs given its 'Open' status? I have Java programming experience and would like to contribute to open source. Thanks.

**Comment by levimatheri** [comment]
*up for grabs

**Comment by nsoft** [comment]
Yup, some there are some contribution guidelines here: https://github.com/nsoft/jesterj/blob/master/CONTRIBUTING.md

**Comment by spikerheado1234** [comment]
Hi, is this issue still open? If so, I would like to give it a shot.

**Comment by nsoft** [comment]
Yup, still open

**Comment by shauryapatel27** [comment]
Hi if this issue is still open, I would like to learn about the project and work on this one. 
Please let me know if this is open and if I can work on this.

**Comment by nsoft** [comment]
Go right ahead :)
